sites/public/pages/applications/review/confirmation.tsx

```diff
@@ -33,25 +33,34 @@ const ApplicationConfirmation = () => {
 
   const imageUrl = imageUrlFromListing(listing, parseInt(process.env.listingPhotoSize))
 
-  const reviewOrder = useMemo(() => {
-    if (listing) {
-      if (listing.reviewOrderType == ListingReviewOrder.lottery) {
-        const lotteryEvent = getLotteryEvent(listing)
-        const lotteryText = []
+  const content = useMemo(() => {
+    let lotteryEvent: ListingEvent
+    const lotteryText = []
+
+    switch (listing?.reviewOrderType) {
+      case ListingReviewOrder.firstComeFirstServe:
+        return {
+          text: t("application.review.confirmation.whatHappensNext.fcfs"),
+        }
+      case ListingReviewOrder.lottery:
+        lotteryEvent = getLotteryEvent(listing)
```

Comment:
I could be missing it, especially since I don't know what the different colors in the google doc represent. But I don't see any of the text say they still want the lottery date listed.